libs/design/paginator/src/paginator/paginator.component.ts

```diff
@@ -86,6 +75,11 @@ export class DaffPaginatorComponent implements OnChanges {
    */
   @Input() queryParam = 'page';
 
+  /**
+   * The aria label.
+   */
+  @Input() ariaLabel: string;
```

Comment:
Can this be `aria-label` instead? Will it automatically pick up the label using the native attribute?